service/service.go

```diff
@@ -123,7 +131,11 @@ func New(name string) *Service {
 		Router:     mux.NewRouter(),
 		ServeMux:   http.NewServeMux(),
 	}
+
+	s.Status.IsHealthy = true
```

Comment:
I am not sure if this is the proper place for status set? I mean, we have a `Start()` func that spin off service. Is the creation on the `service` proper place for this setup here? Actually service is not ready/health because it's not even started/serving, isn't it?
And btw... I really think that we should get a `chan` out of `service.Start()` to be able to notify/get notified about stuff (like if service actually ready and listen etc, and maybe statuses management should be through it?).

Reply:
The Chan on the start is a separate concern, definitely we should have it. Notice, that I have moved the creation of the stop chan, so you can subscribe to it.

This setting is the default value, you, as a service-based-on-missy developer, should manage it yourself.